.github/workflows/release.yml

```diff
@@ -29,30 +29,31 @@ jobs:
     runs-on: ${{ matrix.os }}
 
     strategy:
+      fail-fast: false
```

Comment:
What is the reason for not failing fast? If the fast builds fail, we can save quite a bit of resources by stopping the slow ones.

Reply:
This was just for debugging convenience when different builds were failing for different reasons so I wanted to see them all (try to) finish - I'll revert this